pkg/generator/kcl/kcl_generator.go

```diff
@@ -72,9 +73,11 @@ func Run(o *generator.Options, stack *projectstack.Stack) (*CompileResult, error
 
 	var result *kcl.KCLResultList
 	if o.IsKclPkg {
-		result, err = api.RunCurrentPkg(&opt.CompileOptions{
+		opts := &opt.CompileOptions{
 			Option: kclpkg.NewOption().Merge(optList...),
-		})
+		}
+		opts.SetPkgPath(o.WorkDir)
```

Comment:
The options include `workDir`, and I understand that there is no need to assign `workDir` to `pkgPath` separately.
@zong-zhe